Datastream/.gitattributes

```diff
@@ -0,0 +1,3 @@
+/*.xml.dist export-ignore
+/tests export-ignore
+/.github export-ignore
```

Comment:
Should we start adding `.repo-metadata.json`, `owlbot.yaml` and `.OwlBot.yaml` to these?

Reply:
Sure, can't hurt.